### PR TITLE
ghost role cooldown

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Adds in a short cooldown (5 minutes) for ghost role spawners upon death.

Notes:
 - Does not affect the first death, unless the death was caused by suicide.
 - Does not affect golem shells, xenobiology spawns or any over antagonist related ghost roles (pirates), this only affects the mapping ones such as lavaland doctors.

The effect is very minor (being only 5 minutes after death) and merely prevents spamming while having almost no impact on players that are not abusing them.e

## Why It's Good For The Game

Prevents one player from taking all of the ghost roles and going to try and fight an ash drake with all 3 lives.

## Changelog
:cl:
add: Adds a 5-minute cooldown to non antagonist related ghost roles if the client has more than 2 deaths or has committed suicide.
code: Replaces spaces in hilbert's hotel with tabs
/:cl:

### CODE REVIEW

File: code/modules/awaymissions/corpse.dm [hunk 1]
```diff
@@ -333,6 +337,8 @@
 	icon_state = "sleeper"
 	short_desc = "You are a space doctor!"
 	assignedrole = "Space Doctor"
+	use_cooldown = TRUE
+	use_cooldown = TRUE
```
Comment (PowerfulBacon) [inline code/modules/awaymissions/corpse.dm]:
```suggestion
	use_cooldown = TRUE // Use cooldown
```

**Comment by qwertyquerty** [comment]
As far as I can tell, someone would have the cooldown if they were cloned and then died again, despite not having done a ghost role

**Comment by PowerfulBacon** [comment]
> 
> 
> As far as I can tell, someone would have the cooldown if they were cloned and then died again, despite not having done a ghost role

Yea it would

**Review by zeskorion — DISMISSED** [review]

File: config/Sage/config.txt [hunk 2]
```diff
@@ -538,3 +538,6 @@ VOTE_AUTOTRANSFER_ENABLED
 VOTE_AUTOTRANSFER_INITIAL 54000
 ## Time (in deciseconds) between subsequent transfer votes. Default: 30 minutes
 VOTE_AUTOTRANSFER_INTERVAL 18000
+
+## Ghost role cooldown time after death
```
Comment (ike709) [inline config/Sage/config.txt]: Specify the unit of time.

File: config/config.txt [hunk 3]
```diff
@@ -541,3 +541,6 @@ VOTE_AUTOTRANSFER_ENABLED
 VOTE_AUTOTRANSFER_INITIAL 54000
 ## Time (in deciseconds) between subsequent transfer votes. Default: 30 minutes
 VOTE_AUTOTRANSFER_INTERVAL 18000
+
+## Ghost role cooldown time after death
```
Comment (ike709) [inline config/config.txt]: Ditto

**Review by ike709 — APPROVED** [review]
@Crossedfall Config

**Review by Crossedfall — APPROVED** [review]
Config applied

**Pull request merged**